pkg/backend/display/testdata/not-truncated/failed-stacks-dont-rewrite-operations.json.interactive-100x80-plain.stdout.txt

```diff
@@ -0,0 +1,28 @@
+View in Browser (Ctrl+O): link
+
+<%hide-cursor%>
     Type                               Name      Status                  Info<%clear-to-end%>
+     pulumi:pulumi:Stack                test-dev                          1 message<%clear-to-end%>
+ --  ├─ pulumi-python:dynamic:Resource  test-r    deleted original        <%clear-to-end%>
+ ~   └─ pulumi-python:dynamic:Resource  test-x    **updating failed**     <%clear-to-end%>
+<%clear-to-end%>
+Changes:
+  pulumi-python:dynamic:Resource (test-r):
+    --pulumi-python:dynamic:Resource: (delete-replaced)
+        [id=foo]
+        [urn=urn:pulumi:dev::test::pulumi-python:dynamic:Resource::test-r]
+
+  pulumi-python:dynamic:Resource (test-x):
+    ~ pulumi-python:dynamic:Resource: (update)
+        [id=quux2]
+        [urn=urn:pulumi:dev::test::pulumi-python:dynamic:Resource::test-x]
+
+Diagnostics:
+  pulumi:pulumi:Stack (test-dev):
+    Configuration:
+
+Resources:
+    1 unchanged
+
+Duration: 8s
```

Comment:
Re: `8s` - what happens to this test when run on a different hardware?

Reply:
The timestamps are calculated from the events:

https://github.com/pulumi/pulumi/blob/8baa8386f194849360b524e091b641db3c04f2f0/pkg/backend/display/testdata/not-truncated/failed-stacks-dont-rewrite-operations.json#L1-L9

Reply:
Is 8s part of test validation? E.g. if the test takes 9s on a different machine, will the test fail? I supposed not, but I don't see how this number is ignored.

Reply:
No, it will always print 8s as long as it's the difference between the timestamps in line 0 and line 9, I believe.